index.js

```diff
@@ -7,12 +7,128 @@ function ask(questionText) {
   });
 }
 
-start();
+//random number generator
+function randomInt(maxNum, minNum) {
+  return Math.floor(minNum + Math.random() * (maxNum - minNum + 1));
+}
+
+// calculate guessed number
+function calcInt(maxNum, minNum) {
+  return Math.floor(((maxNum - minNum) / 2) + minNum)
+}
+
+go();
+// initialize global game variables
+let highNum = 5;
```

Comment:
This highNum only allows the computer to guess from 1 to 5, change to 100 for the default and you'll be good.  Also, is there a way you could ask the user what the highNum should be?